core/src/main/java/io/ddf/etl/TransformationHandler.java

```diff
@@ -114,26 +112,45 @@ public DDF transformUDF(String RExp) throws DDFException {
    * @return "(arrtime - crsarrtime) as foobar, (distance / airtime) as speed
    */
 
-  public static String RToSqlUdf(String RExp, List<String> existingColumns) {
+  public static String RToSqlUdf(String RExp, List<String> selectedColumns, List<Column> existingColumns) {
```

Comment:
What if we have expressions with functions existing in R but not in Hive? Don't we support native R expression using Rserve anymore?